CHANGELOG.md

```diff
@@ -6,26 +6,62 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [v17.0.0] 2023-08-09
+
+### MIGRATION notes
+
+- This release updates the `hashicorp/aws` provider required by Cumulus to `~> 5.0`
+  which in turn requires updates to all modules deployed with Core in the same stack
+  to use a compatible provider version.
+- This update is *not* compatible with prior stack states - Terraform will not
+  allow redeployment of a prior version of Cumulus using an older version of
+  the provider.  Please be sure to validate the install changeset is what you
+  expect prior to upgrading to this version.
+- Migration forward to v17 from v16+ is strongly
+  recommended, as the changeset from the v16.x.y baselines is relatively minimal
+  and has been tested.
+- Upgrading Cumulus to v17 from prior versions should only require the usual
+  terraform init/apply steps.  As always **be sure** to inspect the `terraform plan` or
+  `terraform apply` changeset to ensure the changes between providers are what
+  you're expecting for all modules you've chosen to deploy with Cumulus
+
+### Notable Changes
+
+- **CUMULUS-3258**
+  - @cumulus/api is now compatible *only* with Orca >= 8.1.0.    Prior versions of
+    Orca are not compatible with Cumulus 17+
+  - Updated all hashicorp terraform AWS provider configs to ~> 5.0
+    - Upstream/downstream terraform modules will need to utilize an AWS provider
+      that matches this range
+
+## Changed
+
+- **CUMULUS-3258**
+  - Update @cumulus/api/lib/orca/getOrcaRecoveryStatusByGranuleCollection
+    to @cumulus/api/lib/orca/getOrcaRecoveryStatusByGranuleIdAndCollection and
+    add collectionId to arguments to support Orca v8+ required use of
+    collectionId
+  - Update all Core integration tests/integrations to be compatible with Orca >=
+    v8.1.0 only
+  - Updated all terraform AWS providers to ~> 5.0
+
 ## Fixed
 
 - **CUMULUS-3319**
   - Removed @cumulus/api/models/schema and changed all references to
     @cumulus/api/lib/schema in docs and related models
   - Removed @cumulus/api/models/errors.js
+  - Updated API granule write logic to cause postgres schema/db write failures on an individual granule file write to result in a thrown error/400 return instead of a 200 return and a 'silent' update of the granule to failed status.
```

Comment:
This isn't a framework write so may not be applicable but this could be a part of the conversation with NSIDC. I wonder if it's in the "silent failure" bucket.

Reply:
Possibly.    User workflows that work within the CNM pattern tend to use the framework granule writes -- and while that logic has possible issues related to user expectations, the *granule* should always wind up 'failed', which doesn't track to that ticket iirc.